### DISCUSSION TITLE
Generate intersection scenario

### DISCUSSION THREAD

**Discussion by SExpert12** [carla-simulator/carla#6536] [Q&A]
Carla: 0.9.14
Ubuntu : 20.04 ( focal )
Sumo: 1.17.0

Hello, everyone.
I want to generate an intersection scenario as shown in figure in  CARLA simulator. 
Number of approaches - Four approaches
Number of lane per approach- 3 outgoing(1 for left-turn, 1 for right-turn, 1 for straight movement )- 3 incoming- total 6 lanes
How to generate this in CARLA simulator?

Can I import Sumo simulator scenario in CARLA simulator.

I also check all towns for this kind of intersection except Town11 and Town12 (as they are not getting loaded)

Thanks in advance.

![inter_control](https://github.com/carla-simulator/carla/assets/113903215/731d5357-59e5-43b9-8600-d28708647ce4)

**Comment by seedoilz** [comment]
Did you generate the .xodr file of this map? I wanted to use this map.

**Reply by SExpert12** [reply]
You can ask for roadRunner trial version and import it in Carla simulator

**Reply by SExpert12** [reply]
I think in carla 0.9.15 we can create our map. but you need to check it.
